http_cache/esi.rst

```diff
@@ -126,7 +126,7 @@ matter), Symfony uses the standard ``render`` helper to configure ESI tags:
 
     .. code-block:: twig
 
-        {# app/Resources/views/static/about.html.twig #}
+        {# templates/static/about.html.twig #}
 
         {# you can use a controller reference #}
         {{ render_esi(controller('AppBundle:News:latest', { 'maxPerPage': 5 })) }}
```

Comment:
`App\Controller\NewsController::latestAction` use FQNC notation everywhere?

Reply:
Let's change that in a manual PR as well (not so easy to automate)